src/commands/create.js

```diff
@@ -0,0 +1,265 @@
+			default: '.',
+			requiresArg: true
+		},
+		name: {
+			description: nameDesc
+		},
+		force: {
+			alias: 'f',
+			description: forceDesc,
+			type: 'boolean',
+			default: false
+		},
+		yarn: {
+			description: yarnDesc,
+			type: 'boolean',
+			default: false
+		},
+		git: {
+			description: gitDesc,
+			type: 'boolean',
+			default: false
+		},
+		install: {
+			alias: 'i',
+			description: installDesc,
+			type: 'boolean',
+			default: true
+		},
+		template: {
+			description: templateDesc
+		},
+		dest: {
+			description: destDesc
+		}
+	},
+
+	async handler(argv) {
+		// Prompt if incomplete data
+		if (!argv.dest || !argv.template) {
+			warn('Insufficient command arguments! Prompting...');
+			info('Alternatively, run `zimlet create --help` for usage info.');
+
+			let questions = isMissing(argv);
+			let response = await prompt(questions);
+			Object.assign(argv, response);
+		}
+
+		let cwd = resolve(argv.cwd);
+		argv.dest = argv.dest || dirname(cwd);
+		let isYarn = argv.yarn && hasCommand('yarn');
+		let target = resolve(cwd, argv.dest);
+		let exists = isDir(target);
+
+		if (exists && !argv.force) {
+			return error('Refusing to overwrite current directory! Please specify a different destination or use the `--force` flag', 1);
+		}
+
+		if (exists && argv.force) {
+			if (! await prompt({
+				type: 'confirm',
+				name: 'enableForce',
+				message: `You are using '--force'. Do you wish to continue and overwrite ${target}?`,
+				default: false
+			})) {
+				return error('Refusing to overwrite current directory!', 1);
+			}
+		}
+
+		let repo = argv.template;
+		if (!repo.includes('/')) {
+			repo = `${ORG}/${repo}`;
+			info(`Fetching ${argv.template} template from ${repo}...`);
+		}
+
+		// Use `--name` value or `dest` dir's name
+		argv.name = argv.name || argv.dest;
+
+		let { errors } = isValidName(argv.name);
+		if (errors) {
+			errors.unshift(`Invalid package name: ${argv.name}`);
+			return error(errors.map(capitalize).join('\n  ~ '), 1);
+		}
+
+		// Attempt to fetch the `template`
+		let archive = await gittar.fetch(repo).catch(err => {
+			err = err || { message: 'An error occured while fetching template.' };
+			return error(err.code === 404 ? `Could not find repository: ${repo}` : err.message, 1);
+		});
+
+		let spinner = ora({
+			text: 'Creating project',
+			color: 'magenta'
+		}).start();
+
+		// Extract files from `archive` to `target`
+		// TODO: read & respond to meta/hooks
+		let keeps=[];
+		await gittar.extract(archive, target, {
+			strip: 2,
+			filter(path, obj) {
+				if (path.includes('/template/')) {
+					obj.on('end', () => {
+						if (obj.type === 'File' && !isMedia(obj.path)) {
+							keeps.push(obj.absolute);
+						}
+					});
+					return true;
+				}
+			}
+		});
+
+		if (keeps.length) {
+			let dict = new Map();
+
+			// TODO: concat author-driven patterns
+			//globally replace instances of {{name}} with value of name arg
+			['name'].forEach(str => {
```

Comment:
The template replacement seems to be failing, I'm seeing a couple untransformed `{{name}}` strings in a newly generated project:

```sh
$: grep -R -e '{{name}}' src
src/constants/index.js:export const SLUG = '{{name}}';
src/index.js:                   <Text id="{{name}}.menuItem" />
src/components/app/style.less:.{{name}} {
src/intl.json:  "{{name}}": {
```

Reply:
- [ ] fix not all `{{name}}` references being replace